pkgs/development/chez-modules/chez-matchable/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit, chez }:
+
+stdenv.mkDerivation {
+  pname = "chez-matchable";
+  version = "1.0";
+
+  src = fetchgit {
```

Comment:
maybe use `fetchFromGitHub` here

Reply:
What's the difference?

Reply:
fetchFromGitHub fetches the tarball from github, which makes downloads faster

Reply:
Sorry for the delay, done! (assuming it looks ok now?)